nprlite.go

```diff
@@ -150,7 +150,7 @@ func parsearticle(body io.Reader) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	pieces := strings.Split(string(buf), "<p><a href=\"/\">Home</a></p>")
+	pieces := regexp.MustCompile(`<p><a href="/">Home</a>.*?</p>`).Split(string(buf), 2)
```

Comment:
The `?` behind the `*` qualifies that it shouldn't be greedy, so it will stop matching when the first `</p>` is discovered, rather than the last.